cmake/modules/Findbabeltrace.cmake

```diff
@@ -16,7 +16,7 @@ find_program(BABELTRACE_EXECUTABLE
   NAMES babeltrace babeltrace-ctf)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(BABELTRACE DEFAULT_MSG
+find_package_handle_standard_args(babeltrace DEFAULT_MSG
```

Comment:
see https://cmake.org/cmake/help/v3.0/module/FindPackageHandleStandardArgs.html

> In this mode, the name of the result-variable can be set either to either <UPPERCASED_NAME>_FOUND or <OriginalCase_Name>_FOUND using the FOUND_VAR option. Other names for the result-variable are not allowed. So for a Find-module named FindFooBar.cmake, the two possible names are FooBar_FOUND and FOOBAR_FOUND. It is recommended to use the original case version. If the FOUND_VAR option is not used, the default is <UPPERCASED_NAME>_FOUND.

so i think it's fine to use either upper case or lower case or camel case...